src/LibraryManager.Contracts/ExitCode.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Web.LibraryManager.Contracts
+{
+    /// <summary>
+    /// Exit codes of libman tool
+    /// </summary>
+    public enum ExitCode
```

Comment:
This needn't be in the Contracts library.  Contracts is for establishing the interfaces that are common between each different host (MSBuild task, VSIX, CLI tool).  Since this is specific to the CLI tool, it should be in the libman project.